xrpl4j-binary-codec/src/main/java/org/xrpl/xrpl4j/codec/binary/XrplBinaryCodec.java

```diff
@@ -122,6 +126,7 @@ public String encodeForMultiSigning(String json, String xrpAccountId) throws Jso
    * @param json String containing JSON to be encoded.
    *
    * @return The binary encoded JSON in hexadecimal form.
+   *
```

Comment:
This formatting seems different than what we have elsewhere. We usually don't have an empty line between `@return` and `@throws`. Not a huge deal but we should double check your intellij settings to reduce PR noise in the future.

Reply:
It was part of my formatter, but I forgot to reformat code before pushing. Thanks for catching this.